include/render/VolumeRenderer.h

```diff
@@ -33,32 +32,33 @@
 #include "Volume.h"
 
 class VolumeRenderer
-	{
-	public:
-		VolumeRenderer(){};
-		virtual ~VolumeRenderer(){};
+{
+public:
+	VolumeRenderer(){};
+	virtual ~VolumeRenderer(){};
+
+	virtual void initGL() = 0;
+	virtual void render(Volume *volume, const glm::mat4 &MV, glm::mat4 &P, float z_scale, GLint colorma, int renderChannel) = 0;
 
-		virtual void initGL() = 0;
-		virtual void render(Volume* volume, const glm::mat4 &MV, glm::mat4 &P, float z_scale, GLint colorma, int renderChannel) = 0;
+	virtual void set_threshold(float threshold) = 0;
+	virtual void set_multiplier(float multiplier) = 0;
+	virtual void set_num_slices(int slices) = 0;
 
-		virtual void set_threshold(float threshold) = 0;
-		virtual void set_multiplier(float multiplier) = 0;
-		virtual void set_numSlices(int slices) = 0;
-	
-		virtual void set_blending(bool useBlending, float alpha, Volume* volume) = 0;
-		virtual void useMultichannelColormap(bool useMulti) = 0;
-	
-		virtual void setClipMinMax(glm::vec3 min_clip, glm::vec3 max_clip) = 0;
+	virtual void set_blending(bool useBlending, float alpha, Volume *volume) = 0;
+	virtual void useMultichannelColormap(bool useMulti) = 0;
 
-		void setClipping(bool isClipping, glm::mat4 * clipPlane)
-		{
-			m_clipping = isClipping;
-			if (m_clipping)
-				m_clipPlane = *clipPlane;
-		};
+	virtual void set_clip_min_max(glm::vec3 min_clip, glm::vec3 max_clip) = 0;
 
-	protected:
-		bool m_clipping;
-		glm::mat4 m_clipPlane; 
+	void set_clipping_plane(bool isClipping, glm::mat4 *clipPlane)
+	{
+		m_clipping = isClipping;
+		if (m_clipping)
+			m_clip_plane = *clipPlane;
```

Comment:
```suggestion
		if (isClipping)
			m_clip_plane = *clipPlane;
```

Reply:
I dont see an issue using the member variable after being set.